app/build.gradle

```diff
@@ -1,35 +1,36 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion ANDROID_TARGET_SDK_VERSION.toInteger()
     defaultConfig {
         applicationId "com.plangrid.android.sqlitefts5example"
-        minSdkVersion 15
-        targetSdkVersion 26
+        minSdkVersion ANDROID_MIN_SDK_VERSION.toInteger()
+        targetSdkVersion ANDROID_TARGET_SDK_VERSION.toInteger()
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-    buildToolsVersion '26.0.2'
+    buildToolsVersion ANDROID_BUILD_TOOLS_VERSION
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:26.1.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    implementation 'io.requery:sqlite-android:3.22.0'
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha4'
+    implementation 'androidx.sqlite:sqlite:2.0.1'
```

Comment:
Why do we need this in addition to requery?

Reply:
This is so this library can see the androidx sqlite support classes (won't build without it)